### PR TITLE
Add support for Lutron Aurora dimmer

### PR DESCRIPTION
**Pull request by TFenby**

### CODE REVIEW

**Comment by TFenby** [comment]
Looks like I'm missing this configure section:
`        configure: (ieeeAddr, shepherd, coordinator, callback) => {
            const ep1 = shepherd.find(ieeeAddr, 1);
            const actions = [
                (cb) => ep1.bind('genLevelCtrl', coordinator, cb),
            ];
            execute(ep1, actions, callback);
        },`

I thought it was working without it, but that may have just been cached in the database? I'm still trying to pick my way through the codebase.

**Comment by Koenkk** [comment]
Can you add that configure section to the pr?

**Comment by TFenby** [comment]
Certainly. Is there anything else I can do to make this more complete? I tried playing with the HASS discovery but I wasn't sure if it was proper to consider it a "light sensor", and there doesn't seem to be a proper "dimmer" object in HASS.

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -3128,6 +3128,13 @@ const converters = {
             };
         },
     },
+    dimmer_passthru_level: {
+        cid: 'genLevelCtrl',
+        type: 'cmdMoveToLevelWithOnOff',
+        convert: (model, msg, publish, options) => {
+            return {action: 'setlevel', level: msg.data.data.level};
```
Comment (Koenkk) [inline converters/fromZigbee.js]: Could you change `level` to `brightness`? This make it's more consistent with other devices (e.g. the Hue remote dimmer).

**Comment by Koenkk** [comment]
It should be exposed as a action sensor.

I've added one remark, if this is fixed we can merge it.

**Comment by TFenby** [comment]
Done! One other question: I've found that twisting this device causes so many MQTT messages to be sent that it actually locks up some of my lights. Should I add some sort of "buffer" to this code so it only sends the brightness after the knob has stopped moving? Or should that be handled in HASS?

**Comment by TFenby** [comment]
Github seems to be getting confused... Let me know if it shows as renamed on your end?
image

**Comment by Koenkk** [comment]
If the frequence is less than 1s, please debounce it (see fromZigbee.js for examples)

**Comment by TFenby** [comment]
That'll definitely need to be done then. New PR incoming (hopefully) today...

**Comment by TFenby** [comment]
Working well over here :)

**Comment by Koenkk** [comment]
Looks good, thanks!

**Pull request merged**